include/range/v3/utility/common_tuple.hpp

```diff
@@ -415,42 +415,42 @@ namespace ranges
                     std::tuple<Ts...>>;
         }
 
-        // common_type for std::pairs
+        // common_type for pairs
         template<typename F1, typename S1, typename F2, typename S2>
         struct common_type<std::pair<F1, S1>, common_pair<F2, S2>>
           : detail::common_type_tuple_like<std::pair<F1, S1>, common_pair<F2, S2>,
-                meta::quote<std::pair>>
+                meta::quote<detail::make_common_pair>>
```

Comment:
`make_foo` encapsulates the "choose `foo` if there are reference members and `std::foo` otherwise" mechanic. I also vaguely recall there being a reason to favor the `std` flavors, so I'm trying to be conservative with the change.